www/src/pages-markdown/download.md

```diff
@@ -4,26 +4,26 @@ date: "2020-02-01"
 path: "/download"
 header: true
 ---
- - [Play Now (Firefox and Chrome)](/game)
 
-To run the game locally on your computer, use the following links:
+<div style="text-align: center;">
+    <img src="../assets/play-now.gif" width="50%"/>
+    <h2><a href="/game">[Play Now]</a></h2>
+    <p>(Firefox and Chrome)</p>
+</div>
 
- - [Download for Windows](http://go.hiddenswitch.com/win)
- - [Download for macOS](http://go.hiddenswitch.com/win)
+For users on other browsers, go to our [Discord](https://discord.gg/HmbESh2) and ask for the Steam key.
  
 To join the mobile betas:
 
- - [iOS Beta](https://testflight.apple.com/join/pkMfO2qa)
- - Join the [Spellsource Google Group](https://groups.google.com/forum/#!forum/spellsource-alpha-testers), then follow the instructions in the welcome message to participate in testing.
-
-#### Windows Users
-
-Choose **Extract All**, and double click on **Hidden Switch Launcher**. This will download the latest version of Spellsource. 
-
 #### iOS Users
 
-You will be prompted to join our TestFlight group, which is Apple's testing service. You will remain anonymous when you join Testflight this way.
+You can find the iOS Beta [here](https://testflight.apple.com/join/pkMfO2qa). You will be prompted to join our TestFlight group, which is Apple's testing service. You will remain anonymous when you join Testflight this way.
 
 #### Android Users
 
+Join the [Spellsource Google Group](https://groups.google.com/forum/#!forum/spellsource-alpha-testers), then follow the instructions in the welcome message to participate in testing.
 You must be logged into Google Play to join the testing program. Your e-mail will be shared with the developers. 
+
+#### Windows Users
+
+Join the [Spellsource Google Group](https://groups.google.com/forum/#!forum/spellsource-alpha-testers), then follow the instructions in the welcome message to participate in testing.
```

Comment:
spellsource google group under windows is wrong

Reply:
go to the discord and ask for a steam key is the procedure for Windows and macOS users